src/BookPage.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import Nav from './components/Nav';
+import ActionNameButton from './components/BookComponent/ActionNameButton';
+import Percentage from './components/BookComponent/Percentage';
+import style from './BookPage.module.css';
+import AddBook from './components/BookComponent/AddBook';
+
+function BookPage() {
+  return (
+    <>
+      <Nav />
```

Comment:
[OPTION] The navbar component should not be added here. It should be added to the app.js file so that it can be displayed on both the book and categories page. This will help you to achieve best JS practices as well as making it easier to navigate through the website. :wink: